.github/workflows/dismiss-approvals.yml

```
@@ -0,0 +1,32 @@
+name: Dismiss PR approvals on non-maintainer pushes
```

I think this needs a better name, because it's always shown in the list of checks.

```suggestion
name: Check pusher permissions
```